core/runtime.rs

```diff
@@ -194,6 +194,15 @@ pub struct RuntimeOptions {
   pub create_params: Option<v8::CreateParams>,
 }
 
+/// Status of pending tasks in the event loop
+#[derive(Debug, PartialEq, Eq)]
+pub enum PendingState {
+  Active,
+  Idle,
+  BlockedModule,
```

Comment:
IMO this is misleading, they are not blocked but rather "unresolved"

Reply:
Renamed